benchmark_runner/common/oc/oc.py

```diff
@@ -148,7 +148,7 @@ def get_operator_version(self, namespace):
         version = self.run(f"{self._cli} get csv -n {namespace} -o jsonpath='{{.items[0].spec.version}}'")
         return '.'.join(version.split('.')[:2])
 
-    def wait_for_operator_installation(self, operator: str, version: str, namespace: str, timeout: int = int(environment_variables.environment_variables_dict['timeout'])):
+    def wait_for_operator_installation(self, operator: str, version: str, namespace: str, timeout: int =  SHORT_TIMEOUT):
```

Comment:
The PR does not describe the issue that this is going to fix.

Reply:
Right, but why does it need to be changed back?

Reply:
The issue was not the operator wait time, there was issue in the operator version secret, I have fixed it in Jenkinsfile.

Reply:
Then please fix the description of the PR.

Reply:
Done